src/app/screens/speedUpTransaction/index.tsx

```diff
@@ -0,0 +1,541 @@
+import ledgerConnectDefaultIcon from '@assets/img/ledger/ledger_connect_default.svg';
+import ledgerConnectBtcIcon from '@assets/img/ledger/ledger_import_connect_btc.svg';
+import { ledgerDelay } from '@common/utils/ledger';
+import BottomModal from '@components/bottomModal';
+import ActionButton from '@components/button';
+import LedgerConnectionView from '@components/ledger/connectLedgerView';
+import TopRow from '@components/topRow';
+import useTransaction from '@hooks/queries/useTransaction';
+import useBtcClient from '@hooks/useBtcClient';
+import useSeedVault from '@hooks/useSeedVault';
+import useWalletSelector from '@hooks/useWalletSelector';
+import Transport from '@ledgerhq/hw-transport-webusb';
+import { CarProfile, Lightning, RocketLaunch, ShootingStar } from '@phosphor-icons/react';
+import { getBtcFiatEquivalent, rbf } from '@secretkeylabs/xverse-core';
+import { Transport as TransportType } from '@secretkeylabs/xverse-core/ledger/types';
+import type { RecommendedFeeResponse } from '@secretkeylabs/xverse-core/types/api/esplora';
+import { currencySymbolMap } from '@secretkeylabs/xverse-core/types/currency';
```

Comment:
imports should come from root `@secretkeylabs/xverse-core`

Reply:
Yeah, I just updated it. The only one that's not exported from the root is `RecommendedFeeResponse`:
<img width="870" alt="Screenshot 2023-11-29 at 12 27 04" src="https://github.com/secretkeylabs/xverse-web-extension/assets/36603049/9ff0830c-8a46-4b2d-a417-7be5444c1036">

Reply:
Ah bleak. I'll add it to the core PR 👍

Reply:
Ok, it's in

Reply:
Thanks, I updated it